services/order.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime.date = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+        for ticket in tickets:
+            movie_session = MovieSession.objects.get(

```

Comment:
Use `movie_session_id`, as the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency) suggests.